saffron/e2e-test.sh

```diff
@@ -21,9 +22,7 @@ if [ ! -f "$INPUT_FILE" ]; then
 fi
 
 # Compute commitment and capture last line
-COMMITMENT=$(cargo run --release --bin saffron compute-commitment -i "$INPUT_FILE" $SRS_ARG | tee /dev/stderr | tail -n 1)
-
-
+COMMITMENT=$(cargo run --release --bin saffron compute-commitment -i "$INPUT_FILE" -o "$COMMITMENT_FILE" $SRS_ARG | tee /dev/stderr | tail -n 1)
```

Comment:
Why `tee /dev/stderr`?

Reply:
This is so that we can capture the final output of the command in the `COMMITMENT` variable as well as display the logs that it generates in the console while it is running

Reply:
Ah, I see. I thought you wanted to discard the logs and only get the commitment value.